repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -366,6 +367,17 @@ const createProductDetails = async (
         productDetailsElements.push({ id: 'time-restriction', name: 'Only valid during term time', content: ['Yes'] });
     }
 
+    if ('cap' in ticket && ticket.cap) {
```

Comment:
what if we have no caps but wanted to add some? I think this check should see if we have already created caps.

Reply:
this is for the edit functionality and do we need to show the N/A

Reply:
I agree with Alaina - if you look at the salesConfirmation.tsx file, we have 

`  if (['single', 'return', 'flatFare'].includes(fareType) && hasCaps) {
        confirmationElements.push({
            name: 'Cap',
            content: selectedCap?.capDetails.name || 'N/A',
            href: 'selectCaps',
        });
    }`

So we need the same thing inside the if statement here

Reply:
Can we also only display this on test + local environment